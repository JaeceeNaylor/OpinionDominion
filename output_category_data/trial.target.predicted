SentenceID: 1090587:1
Category: NULL#NULL

SentenceID: 1090587:2
Category: NULL#NULL

SentenceID: 1090587:2
Category: NULL#NULL

SentenceID: 1090587:2
Category: NULL#NULL

SentenceID: 1090587:3
Category: NULL#NULL

SentenceID: 1661043:0
Category: NULL#NULL

SentenceID: 1661043:1
Category: NULL#NULL

SentenceID: 1661043:2
Category: NULL#NULL

SentenceID: 1661043:3
Category: NULL#NULL

SentenceID: 1661043:4
Category: NULL#NULL

SentenceID: 1661043:5
Category: NULL#NULL

SentenceID: 1349391:0
Category: NULL#NULL

SentenceID: 1349391:0
Category: NULL#NULL

SentenceID: 1349391:1
Category: NULL#NULL

SentenceID: 1349391:1
Category: NULL#NULL

SentenceID: 1349391:1
Category: NULL#NULL

SentenceID: 1349391:1
Category: FOOD#PRICES

SentenceID: 1349391:2
Category: NULL#NULL

SentenceID: 1357554:0
Category: NULL#NULL

SentenceID: 1357554:2
Category: NULL#NULL

SentenceID: 1357554:3
Category: NULL#NULL

SentenceID: 1357554:3
Category: NULL#NULL

SentenceID: 1357554:4
Category: RESTAURANT#GENERAL

SentenceID: 1357554:5
Category: RESTAURANT#GENERAL

SentenceID: 1730014:1
Category: NULL#NULL

SentenceID: 1730014:1
Category: NULL#NULL

SentenceID: 1730014:2
Category: NULL#NULL

SentenceID: 1730014:2
Category: NULL#NULL

SentenceID: 1730014:2
Category: NULL#NULL

SentenceID: 1730014:3
Category: NULL#NULL

SentenceID: 1500453:1
Category: NULL#NULL

SentenceID: 1500453:1
Category: NULL#NULL

SentenceID: 1500453:2
Category: NULL#NULL

SentenceID: 1500453:2
Category: NULL#NULL

SentenceID: 1500453:2
Category: NULL#NULL

SentenceID: 1500453:2
Category: FOOD#PRICES

SentenceID: 1500453:3
Category: NULL#NULL

SentenceID: 1500453:3
Category: NULL#NULL

SentenceID: 1500453:4
Category: NULL#NULL

SentenceID: 1500453:4
Category: NULL#NULL

SentenceID: 1500453:4
Category: NULL#NULL

SentenceID: 1500453:4
Category: NULL#NULL

SentenceID: 1500453:5
Category: NULL#NULL

SentenceID: 505535:0
Category: RESTAURANT#GENERAL

SentenceID: 505535:0
Category: RESTAURANT#PRICES

SentenceID: 505535:1
Category: NULL#NULL

SentenceID: 505535:1
Category: NULL#NULL

SentenceID: 1016296:0
Category: RESTAURANT#GENERAL

SentenceID: 1016296:1
Category: NULL#NULL

SentenceID: 1016296:2
Category: NULL#NULL

SentenceID: 1016296:3
Category: NULL#NULL

SentenceID: 1016296:4
Category: NULL#NULL

SentenceID: 1016296:4
Category: NULL#NULL

SentenceID: 1016296:4
Category: NULL#NULL

SentenceID: 1016296:5
Category: NULL#NULL

SentenceID: Z#3:0
Category: NULL#NULL

SentenceID: Z#3:0
Category: NULL#NULL

SentenceID: Z#3:1
Category: NULL#NULL

SentenceID: Z#3:2
Category: NULL#NULL

SentenceID: Z#3:2
Category: NULL#NULL

SentenceID: Z#3:3
Category: NULL#NULL

SentenceID: PP#9:0
Category: NULL#NULL

SentenceID: PP#9:1
Category: NULL#NULL

SentenceID: PP#9:2
Category: NULL#NULL

SentenceID: PP#9:2
Category: NULL#NULL

SentenceID: PP#9:3
Category: RESTAURANT#GENERAL

