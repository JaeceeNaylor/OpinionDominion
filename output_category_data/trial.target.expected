SentenceID: 1090587:0
Category: NULL

SentenceID: 1090587:1
Category: LOCATION#GENERAL

SentenceID: 1090587:2
Category: SERVICE#GENERAL

SentenceID: 1090587:2
Category: FOOD#QUALITY

SentenceID: 1090587:2
Category: FOOD#PRICES

SentenceID: 1090587:3
Category: FOOD#QUALITY

SentenceID: 1661043:0
Category: FOOD#QUALITY

SentenceID: 1661043:1
Category: FOOD#QUALITY

SentenceID: 1661043:2
Category: FOOD#QUALITY

SentenceID: 1661043:3
Category: FOOD#QUALITY

SentenceID: 1661043:4
Category: AMBIENCE#GENERAL

SentenceID: 1661043:5
Category: SERVICE#GENERAL

SentenceID: 1349391:0
Category: FOOD#QUALITY

SentenceID: 1349391:0
Category: SERVICE#GENERAL

SentenceID: 1349391:1
Category: FOOD#QUALITY

SentenceID: 1349391:1
Category: SERVICE#GENERAL

SentenceID: 1349391:1
Category: SERVICE#GENERAL

SentenceID: 1349391:1
Category: FOOD#QUALITY

SentenceID: 1349391:2
Category: RESTAURANT#GENERAL

SentenceID: 1349391:3
Category: NULL

SentenceID: 1357554:0
Category: DRINKS#STYLE_OPTIONS

SentenceID: 1357554:1
Category: NULL

SentenceID: 1357554:2
Category: FOOD#QUALITY

SentenceID: 1357554:3
Category: SERVICE#GENERAL

SentenceID: 1357554:3
Category: SERVICE#GENERAL

SentenceID: 1357554:4
Category: AMBIENCE#GENERAL

SentenceID: 1357554:5
Category: RESTAURANT#GENERAL

SentenceID: 1730014:0
Category: NULL

SentenceID: 1730014:1
Category: SERVICE#GENERAL

SentenceID: 1730014:1
Category: FOOD#QUALITY

SentenceID: 1730014:2
Category: FOOD#QUALITY

SentenceID: 1730014:2
Category: FOOD#QUALITY

SentenceID: 1730014:2
Category: FOOD#QUALITY

SentenceID: 1730014:3
Category: RESTAURANT#GENERAL

SentenceID: 1730014:4
Category: NULL

SentenceID: 1500453:0
Category: NULL

SentenceID: 1500453:1
Category: AMBIENCE#GENERAL

SentenceID: 1500453:1
Category: RESTAURANT#GENERAL

SentenceID: 1500453:2
Category: FOOD#QUALITY

SentenceID: 1500453:2
Category: FOOD#QUALITY

SentenceID: 1500453:2
Category: FOOD#QUALITY

SentenceID: 1500453:2
Category: FOOD#QUALITY

SentenceID: 1500453:3
Category: FOOD#QUALITY

SentenceID: 1500453:3
Category: FOOD#QUALITY

SentenceID: 1500453:4
Category: AMBIENCE#GENERAL

SentenceID: 1500453:4
Category: FOOD#QUALITY

SentenceID: 1500453:4
Category: SERVICE#GENERAL

SentenceID: 1500453:4
Category: AMBIENCE#GENERAL

SentenceID: 1500453:5
Category: RESTAURANT#GENERAL

SentenceID: 505535:0
Category: AMBIENCE#GENERAL

SentenceID: 505535:0
Category: RESTAURANT#PRICES

SentenceID: 505535:1
Category: FOOD#QUALITY

SentenceID: 505535:1
Category: FOOD#QUALITY

SentenceID: 505535:2
Category: NULL

SentenceID: 1016296:0
Category: RESTAURANT#GENERAL

SentenceID: 1016296:1
Category: SERVICE#GENERAL

SentenceID: 1016296:2
Category: SERVICE#GENERAL

SentenceID: 1016296:3
Category: FOOD#QUALITY

SentenceID: 1016296:4
Category: FOOD#QUALITY

SentenceID: 1016296:4
Category: FOOD#QUALITY

SentenceID: 1016296:4
Category: RESTAURANT#MISCELLANEOUS

SentenceID: 1016296:5
Category: RESTAURANT#GENERAL

SentenceID: Z#3:0
Category: FOOD#QUALITY

SentenceID: Z#3:0
Category: AMBIENCE#GENERAL

SentenceID: Z#3:1
Category: AMBIENCE#GENERAL

SentenceID: Z#3:2
Category: FOOD#QUALITY

SentenceID: Z#3:2
Category: FOOD#PRICES

SentenceID: Z#3:3
Category: DRINKS#QUALITY

SentenceID: PP#9:0
Category: RESTAURANT#GENERAL

SentenceID: PP#9:1
Category: FOOD#QUALITY

SentenceID: PP#9:2
Category: SERVICE#GENERAL

SentenceID: PP#9:2
Category: FOOD#QUALITY

SentenceID: PP#9:3
Category: RESTAURANT#GENERAL

